GhostNetwork.Content.Api/Controllers/CommentsController.cs

```diff
@@ -45,6 +46,29 @@ public async Task<ActionResult<Comment>> CreateAsync([FromBody] CreateCommentMod
             return BadRequest(domainResult.ToProblemDetails());
         }
 
+        /// <summary>
+        /// Update comment content
+        /// </summary>
+        /// <param name="commentId">Existing comment ID</param>
+        /// <param name="content">New content</param>
+        /// <returns>Updated comment</returns>
+        [HttpPut("{commentId}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<Comment>> UpdateAsync([FromRoute] string commentId, [FromBody][Required] string content)
+        {
+            var (domainResult, (comment, updateCount)) = await commentService.UpdateAsync(commentId, content);
+
+            if (!domainResult.Successed)
+            {
+                return BadRequest();
+            }
+
+            return comment == null ? NotFound() : updateCount == 0 ? NoContent() : Ok(comment);
```

Comment:
What is the case for `updateCount == 0` case?
In case there is nothing to update will be returned **NotFound**, in case of update should be returned **NoContent** OR **Ok** NOT both of them

Reply:
Thanks, I take a mistake while reading the http documentation